src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayRentCost = 40;
+  const threeDayRentDiscount = 20;
+  const sevenDayRentDiscount = 50;
+  const smallTermRent = 3;
+  const longTermRent = 7;
+  const totalRent = oneDayRentCost * days;
```

Comment:
I think, thats okey, but as I see in other works, you should change all variables name from `oneDayRentCost` to `ONE_DAY_RENT_COST` for example.

Reply:
I think that camelCase variables are ok too, but I changed them as you said)